bin/plugin/commands/changelog.js

```diff
@@ -33,6 +33,47 @@ const manifest = require( '../../../package.json' );
  * @property {string}  milestone Milestone title.
  */
 
+/**
+ * Changelog normalization function, returning a string to use as title, or
+ * undefined if entry should be omitted.
+ *
+ * @typedef {(text:string,issue:IssuesListForRepoResponseItem)=>string|undefined} WPChangelogNormalization
+ */
+
+/**
+ * Mapping of label names to grouping heading text to be used in release notes,
+ * intended to be more readable in the context of release notes. Also used in
+ * merging multiple related groupings to a single heading.
+ *
+ * @type {Record<string,string>}
+ */
+const LABEL_GROUP_TITLES = {
+	Bug: 'Bug Fixes',
+	Regression: 'Bug Fixes',
+	Feature: 'Features',
+	Enhancement: 'Enhancements',
+	'New API': 'New APIs',
+};
+
+/**
```

Comment:
I noticed that I had a "Task" heading when running the script today, I wonder if we should remove this generic label to force us to choose something more meaningful (feature, enhancement...)

Reply:
> I noticed that I had a "Task" heading when running the script today, I wonder if we should remove this generic label to force us to choose something more meaningful (feature, enhancement...)

Interesting question. It does seem like "Task" is used as something of a fallback, when "Bug" or "Enhancement" don't quite fit. It may have been inherited from how Types are assigned in core in [ticket creation](https://core.trac.wordpress.org/newticket) ("defect (bug)", "enhancement", "feature request", "task (blessed)").

In the meantime, 22d8fce remaps "Task" to "Various", so at least it's lumped in with the rest of the miscellany.